ilmercatinodellibro/ilmercatinodellibro
# feat(server+client): wire and update users management

## What it does
This feature implements:
- the server logic (and wiring with the frontend UI) of the filters for the "Manage Users"
- the update of the local cache after new settings are applied by an Admin for a Retail Location
- the visualization of users with unverified e-mails in the list
- some minor UI fixes

## How to test it
- Login as Operator or Admin
- Navigate to `/users-management` and verify the presence of unverified users
- As an Admin, change the retail location's settings and click "save", verify that the settings update the UI accordingly (in particular the setting to enable/disable pay-offs)
